crowbar_framework/spec/controllers/nodes_controller_spec.rb

```diff
@@ -154,6 +154,11 @@
       get :families
       assigns(:families).keys.should include(node.family.to_s)
     end
+
+    it "as json is not acceptable" do
```

Comment:
it was a bugreport by QA, it makes sense to add a test that checks that there is not again a 500. plus its a nice learning excercise on how to write specs..